src/wrappers/themis/react-native-themis/src/index.d.ts

```diff
@@ -9,11 +11,10 @@ export declare function secureCellTokenProtectEncrypt64(symmetricKey64: String,
 export declare function secureCellTokenProtectDecrypt64(symmetricKey64: String, encrypted64: String, token64: String, context?: String): Promise<string>;
 export declare function secureCellContextImprintEncrypt64(symmetricKey64: String, plaintext: String, context: String): Promise<string>;
 export declare function secureCellContextImprintDecrypt64(symmetricKey64: String, encrypted64: String, context: String): Promise<string>;
-export declare function secureMessageSign64(plaintext: String, privateKey64: String, publicKey64: String): Promise<string>;
-export declare function secureMessageVerify64(signed64: String, privateKey64: String, publicKey64: String): Promise<string>;
+export declare function secureMessageSign64(plaintext: String, privateKey64: String): Promise<string>;
```

Comment:
`publicKey64` parameter should be reinstated here to avoid breaking changes.

Reply:
done.